draft-ietf-suit-trust-domains.cddl

```diff
@@ -42,15 +42,15 @@ SUIT_Directive //= (
     suit-directive-unlink, SUIT_Rep_Policy)
 
 suit-delegation = 1
-suit-dependency-resolution = 7
+suit-dependency-resolution = 15
 suit-uninstall = 24
 
 suit-dependencies = 1
 
 suit-dependency-prefix = 1
 
 suit-condition-is-dependency            = 7
-suit-directive-process-dependency       = 18
 suit-directive-set-parameters           = 19
 suit-directive-unlink                   = 33
+suit-directive-process-dependency       = 34
```

Comment:
I think override-multiple and copy-params (from update management) should be 1-byte since they are both explicitly space-saving operations.

Reply:
1-byte key size is desirable, because it may appear multiple times per dependency. (e.g. in install and invoke)
How about this? It is also easy to understand; check then process the dependency.
```
suit-condition-is-dependency = 7
suit-directive-process-dependency = 8
```

Reply:
I agree, that makes sense.

Reply:
I've fixed.